src/test/java/com/faforever/api/clan/ClanControllerIntegrationTest.java

```diff
@@ -35,44 +29,25 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
-@Ignore("This needs to be cleaned up big time.")
+@Import(TestDatabase.class)
+@Transactional
 public class ClanControllerIntegrationTest {
```

Comment:
In the future, or is this required to get this merged?
I want not to touch your `AbstractIntegrationTest` implementation, because maybe you have a better way of doing it ...
Because `AbstractIntegrationTest` is not final and currently I  prefer composition over inheritance in this use case it's fine for me ...

But I aggree some real `basic` common subclass is essential for the future (next step):
https://github.com/FAForever/faf-java-api/issues/72